pages/docs/components/typography/heading.mdx

```diff
@@ -67,8 +67,9 @@ the default theme. This can be seen in the [theme source](https://github.com/cha
 
 ### Truncate heading
 
-Pass the `noOfLines={1}` prop to render an ellipsis when the heading exceeds the
-width of the viewport or `maxWidth` prop.
+If you'd like to truncate the heading after a specific number of lines, pass the
+`noOfLines` prop. This will render an ellipsis when the heading exceeds the width
+of the viewport or `maxWidth` prop.
```

Comment:
```suggestion
of the viewport.
```

Reply:
This is implicit in my opinion.

Reply:
Hmmm, I would say including this might be helpful to people. When I think of viewport I think of the whole screen (maybe that's just me). Including the maxWidth prop suggestion here could help some people get the truncation working faster. Thoughts?

Reply:
Then lets leave it like this :)